### PR TITLE
Add Getting Started for Reliably and Chaos Toolkit

### PR DESCRIPTION
**Pull request by ciaranevans**
# What this PR does

TL;DR: This PR adds the `Explore the impact on your reliability using Chaos Engineering experiments` Getting Started entry which takes the reader through the following:

* Defining an SLO
* Creating a Chaos Engineering experiment
* Running the experiment and viewing its impact on your reliability (both good and bad)

# How I did it

* `docs/getting-started/chaos-engineering/index.md`:
  * Wrote the post? - Does what it says on the tin really
* `src/assets/scss/components/_code.scss`:
  * Added the `venv` style to `language-reliably` to include `(.venv) $ ` in terminal output but to not have it copied
  * Added the `language-console-venv` class to use within `SetupWithCTK.vue`
* `src/components/SetupWithCTK.vue`:
  * Add re-usable component that takes the user through installing Reliably **and** Chaos Toolkit (This might be done in a better way - can I use the `Setup.vue` inside this?)
* `src/data/menu-getting-started.yaml`:
  * Added my doc to the menu
  
# How you can test it

Checkout the repo, get setup for development then run:

```bash
$ npx gridsome develop
```

And browse to: http://localhost:8080/getting-started/chaos-engineering/

### CODE REVIEW

File: docs/getting-started/chaos-engineering/index.md [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+So that you don't need to have any infrastructure already available to follow this guide, you'll be defining a Custom Objective. These allow you to send Custom Indicators as Reliably doesn't have any integrations for a 'service' running locally.
+
+For this guide, we're providing you with a ready-made Custom Objective; you should copy this Objective into a file called `reliably.yaml`:
+
+```yaml
+apiVersion: reliably.com/v1
+kind: Objective
+metadata:
+  labels:
+    name: exploring-reliability-guide
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:39]: Maybe use a name that implies what you're surveying? Something like "99.9% of our requests are in the 2xx class"

File: docs/getting-started/chaos-engineering/index.md [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+  window: 1h
+```
+<CopyToClipboard />
+
+### Observe your system
+
+Now that you have an SLO defined, you can observe the reliability of your system:
+
+```reliably
+<span class="token venv"></span>reliably slo sync # Uploads your manifest (reliably.yaml) to Reliably
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:55]: I'd say don't prefix with `venv`, this may be confusing.

File: docs/getting-started/chaos-engineering/index.md [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+<span class="token venv"></span>reliably slo report
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:56]: likewise

File: docs/getting-started/chaos-engineering/index.md [hunk 4]
```diff
@@ -0,0 +1,577 @@
...
+To keep everything local and mimic a service running, you'll use a Chaos Toolkit Experiment to send `Good` indicators to Reliably.
+
+To make things easier, we've provided you with a ready-made Chaos Toolkit Experiment; you should copy this experiment into a file called `healthy-experiment.json`:
+
+```json
+{
+    "version": "1.0.0",
+    "title": "A Healthy Service",
+    "description": "Sends Reliably `Good` Indicators to mimic a healthy service",
+    "tags": [],
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:78]: if unused, you might as well remove this

File: docs/getting-started/chaos-engineering/index.md [hunk 5]
```diff
@@ -0,0 +1,577 @@
...
+  ❌ exploring-reliability-guide                         <span class="token red">90.00%</span>      99.9%  / 1h0m0s          Availability     <span class="token green">✓</span> <span class="token red">✕</span>
+```
+
+Oh dear! You've successfully got a `Bad` service. You can see that your current availability is `90%` and that you've had two indicators pushed as shown by `✓ ✕` under `Trend`
+
+## Using your SLOs as a Steady State Hypothesis
+
+Now that you can observe the reliability of your 'system' and conduct Chaos Engineering experiments against it, it's a good time to further integrate Reliably and Chaos Toolkit!
+
+Earlier, you installed `chaostoolkit-reliably`, this extension of Chaos Toolkit provides functionality that allows you to hook into Reliablys SLOs and use them as your Steady State Hypothesis.
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:345]: missing the apostrophe? `Reliablys SLOs`

File: docs/getting-started/chaos-engineering/index.md [hunk 6]
```diff
@@ -0,0 +1,577 @@
...
+            "provider": {
+                "type": "python",
+                "module": "chaosreliably.slo.probes",
+                "func": "get_objective_results_by_labels",
+                "arguments": {
+                    "labels": {
+                        "name": "exploring-reliability-guide",
+                        "service": "exploring-reliability-guide-service"
+                    },
+                    "limit": 1
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:370]: I would have thought this was the default for that action? therefore not needed explicitely here?

File: docs/getting-started/chaos-engineering/index.md [hunk 7]
```diff
@@ -0,0 +1,577 @@
...
+            },
+            "tolerance": {
+                "type": "probe",
+                "name": "validate-slo-status",
+                "provider": {
+                    "type": "python",
+                    "module": "chaosreliably.slo.tolerances",
+                    "func": "all_objective_results_ok",
+                    "arguments": {}
+                }
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:381]: Interestingly, if our jsonpath tolerance was a bit more powerful, we could probably use it here too.

File: docs/getting-started/chaos-engineering/index.md [hunk 8]
```diff
@@ -0,0 +1,577 @@
...
+            }
+        }
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:515]: I think we ought to add a new probe to the extension and release it again.

That probe would directly coerce these two functions into one. That would make the experiment so much nicer to read.

```json
{
                "type": "python",
                "module": "chaosreliably.slo.probes",
                "func": "slo_is_met",
                "tolerance": true,
                "arguments": {
                    "labels": {
                        "name": "exploring-reliability-guide",
                        "service": "exploring-reliability-guide-service"
                    }
                },
            },
```

You could have an extra argument, such as `last_n_results` defaulting to `1` to the most current or greater for a trend of N results.

File: docs/getting-started/chaos-engineering/index.md [hunk 9]
```diff
@@ -0,0 +1,577 @@
...
+* Running the experiment and viewing its impact on your reliability
+
+<Setup />
+<SetupWithCTK />
+
+## Define your Objective
+
+Check out [Defining and observe your reliability](/docs/getting-started-slos/) for more information on defining SLOs.
+
+So that you don't need to have any infrastructure already available to follow this guide, you'll be defining a Custom Objective. These allow you to send Custom Indicators as Reliably doesn't have any integrations for a 'service' running locally.
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:30]: I would rephrase with a more "positive" spin:

```
These allow you to send Custom Indicators to Reliably and appreciate how indicators can be pushed, not just pulled by the reliably slo agent`.
```

File: docs/getting-started/chaos-engineering/index.md [hunk 10]
```diff
@@ -0,0 +1,577 @@
...
+
+You'll notice that there's no information. That's because Reliably hasn't had any Indicators that match the `indicatorSelector` yet.
+
+## Create a Chaos Engineering experiment
+
+To keep everything local and mimic a service running, you'll use a Chaos Toolkit Experiment to send `Good` indicators to Reliably.
+
+To make things easier, we've provided you with a ready-made Chaos Toolkit Experiment; you should copy this experiment into a file called `healthy-experiment.json`:
+
+```json
```
Comment (Lawouach) [inline docs/getting-started/chaos-engineering/index.md:73]: could we switch to yaml for the entire article? I'd like to keep eye fatigue to a minimum and Reliably deals with yaml mostly anyway.

**Review by Lawouach — COMMENTED** [review]
Generally lgtm. Looking forward your thoughts.

File: docs/getting-started/chaos-engineering/index.md (hunk 1 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: done

File: docs/getting-started/chaos-engineering/index.md (hunk 2 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: done

File: docs/getting-started/chaos-engineering/index.md (hunk 3 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: done

File: docs/getting-started/chaos-engineering/index.md (hunk 4 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: done

File: docs/getting-started/chaos-engineering/index.md (hunk 5 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: Done

File: docs/getting-started/chaos-engineering/index.md (hunk 6 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: done

File: docs/getting-started/chaos-engineering/index.md (hunk 8 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: done

File: docs/getting-started/chaos-engineering/index.md (hunk 9 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md]: done

File: docs/getting-started/chaos-engineering/index.md (hunk 10 above)
Comment (ciaranevans) [reply docs/getting-started/chaos-engineering/index.md:71]: As discussed, with CTK's default being JSON, we'll stick to JSON

**Review by Lawouach — APPROVED** [review]
